shared/src/native-src/filesystem.hpp

```diff
@@ -0,0 +1,6017 @@
+//---------------------------------------------------------------------------------------
```

Comment:
I think you may need to add this to https://github.com/DataDog/dd-trace-dotnet/blob/master/LICENSE-3rdparty.csv ?

Reply:
Thanks for that I did not know

Reply:
done in https://github.com/DataDog/dd-trace-dotnet/pull/2527/commits/a3e613f85c792762eb45d43ae293a94ef4dbc562